notebooks/Fesom2GeoFormat.ipynb

```diff
@@ -0,0 +1,189 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Please note that you need to have the OSGeo (Gdal) Python package installed.\n",
+    "\n",
+    "```bash\n",
+    "\n",
+    " conda install gdal\n",
+    " OR\n",
+    " pip install pygdal==\"`gdal-config --version`.*\"\n",
+    " python\n",
+    " >>>from osgeo import gdal\n",
+    "\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyfesom2 import load_mesh,select_slices,fesom2GeoFormat\n",
+    "import xarray as xr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set Path to the mesh and result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "meshPath  ='tests/data/pi-grid/'\n",
```

Comment:
It should be changed to "../tests/data/pi-grid/" to make it work in the `notebooks` folder.